builder/steps/gen-dockerfile/contents/src/data/Dockerfile

```diff
@@ -0,0 +1,3 @@
+# Dockerfile extending the generic Node image with application files for a
```

Comment:
I've moved them so that there is a `data` directory parallel to `src` containing only a templated `Dockerfile` and the `dockerignore` file.